sdk/eventgrid/eventgrid/README.md

```diff
@@ -10,7 +10,7 @@ Use the client library to:
 
 [Source code](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/eventgrid/eventgrid/) |
 [Package (NPM)](https://www.npmjs.com/package/@azure/eventgrid/v/next) |
-[API reference documentation](https://aka.ms/azsdk-js-eventgrid-ref-docs) |
+[API reference documentation](https://docs.microsoft.com/javascript/api/@azure/eventgrid) |
```

Comment:
Event Grid in the master branch is in preview, so the docs should go the preview version.

```suggestion
[API reference documentation](https://docs.microsoft.com/javascript/api/@azure/eventgrid/?view=azure-node-preview) |
```